IDEAS/Airflow/AHU/Examples/Adsolair58.mo

```diff
@@ -161,14 +160,20 @@ model Adsolair58 "Adsolair58 example model"
     inputType=IDEAS.Fluid.Types.InputType.Stages,
     m_flow_nominal=2,
     heads={0,1e4}) "Supply pump"
-    annotation (Placement(transformation(extent={{-74,-2},{-58,14}})));
+    annotation (Placement(transformation(extent={{-8,-8},{8,8}},
+        rotation=90,
+        origin={-22,18})));
   Fluid.Actuators.Valves.ThreeWayEqualPercentageLinear val(
     redeclare package Medium = MediumWater,
     dpValve_nominal=1e4,
     energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
     tau=30,
     use_inputFilter=false,
-    m_flow_nominal=2)      "Valve for controlling flow rate"
+    m_flow_nominal=2,
+    portFlowDirection_1=Modelica.Fluid.Types.PortFlowDirection.Entering,
+    portFlowDirection_2=Modelica.Fluid.Types.PortFlowDirection.Leaving,
+    portFlowDirection_3=Modelica.Fluid.Types.PortFlowDirection.Entering)
```

Comment:
Is this necessary in general? What if flow reversal?

Reply:
I was debugging an initial algebraic loop in the model that had a numeric Jacobian and made this change in the meantime. The flow cannot reverse in this example.